README.md

```diff
@@ -1217,6 +1217,56 @@ const MyElement = (props, chren) => <div>Hello, {chren()}!</div>;
 
 
 
+#### `vdom.elements (...elements)`
+
+Returns an array of functions that create elements.
+
+```js
+const [ div, p ] = skate.vdom.elements('div', 'p');
+div(p.bind('test'));
+```
+
+Which is the same thing as:
+
+```js
+skate.vdom.element('div', skate.vdom.element.bind(null, 'p', 'test'));
+```
+
+Both of the above would produce:
+
+```html
+<div>
+  <p>test</p>
+</div>
+```
+
+Since this is just syntactic sugar around `skate.vdom.element()`, you can create functions for anything that it accepts as its first argument. This means you can also create functions that wrap stateless functions (function helpers) or even web component constructors:
+
+```js
+const [ div, myFunc, myComponent ] = skate.vdom.elements(
+  'div',
+  (props, chren) => skate.vdom.element('p', props, chren),
+  define('my-component', {})
+);
+div(() => {
+  myFunc(() => {
+    xTest('test');
```

Comment:
That's a good point. I did that and having more than one level seemed to read easier.